templates/app/models/sequelize-model.hbs

```diff
@@ -23,17 +23,25 @@ module.exports = (sequelize, DataTypes) => {
   // This section contains the relationships for this model. See: https://docs.forestadmin.com/documentation/v/v5/reference-guide/relationships#adding-relationships.
   {{modelName}}.associate = (models) => {
     {{#each references as |reference|}}
-    {{../modelName}}.belongsTo(models.{{reference.ref}}, {
+    {{#if reference.isBelongsToMany}}
+    {{../modelName}}.belongsToMany(models.{{reference.ref}}, {
+      through: '{{reference.junctionTable}}',
+      foreignKey: '{{reference.columnName}}',
+      otherKey: '{{reference.otherKey}}',
+    });
+    {{else}}
+    {{../modelName}}.{{reference.association}}(models.{{reference.ref}}, {
       foreignKey: {
         name: '{{reference.foreignKeyName}}',{{#if reference.foreignKeyColumnUnconventional}}
-        field: '{{reference.foreignKey}}',{{/if}}
+        field: '{{reference.columnName}}',{{/if}}
```

Comment:
Looks like `reference.foreignKeyColumnUnconventional` is always true now.
Seems to be a regression.